docs/src/rustc-hacks.md

```diff
@@ -1,9 +1,9 @@
 # Working with `rustc`
-Kani is developed on the top of the rust compiler, which is not distributed on crates.io and it depends on
+Kani is developed on the top of the rust compiler, which is not distributed on [crates.io](https://crates.io/) and it depends on
 bootstrapping mechanisms to properly build its components.
```

Comment:
Is the "bootstrapping mechanisms" part still relevant?

Reply:
I think it is explaining why this dependency does not appear in `Cargo.toml`: The Rust compiler depends on them to properly build, so code extensions need an extra setting for this. I removed the "it" from "it depends on" so that it's clear we're talking about the compiler and not Kani.